integrations/webchat/contributing/template/server/nodejs-express/server.js

```diff
@@ -1,29 +1,32 @@
-#!/usr/bin/env node
-
 /**
- * This file starts up a basic NodeJS server with express, adds some routes, and adds some error handling.
- *
- * ADD A COMMENT HERE ABOUT HOW TO FIND THE SPECIAL CODE YOU HAVE ADDED FOR THIS EXAMPLE.
+ * This file starts up a basic NodeJS server with express, adds some routes, and adds some error handling. This is
+ * mostly boilerplate content that can be adjusted as needed.
  *
  * The server defaults to being available on port 3001.
  */
 
 const { createServer } = require('http');
 
+const cookieParser = require('cookie-parser');
+const cors = require('cors');
 const createError = require('http-errors');
 const express = require('express');
-const cookieParser = require('cookie-parser');
 
-// Import more routes here.
-const apiRouter = require('./routes/api');
+const PORT = 3001;
 
 const app = express();
 
+// Enable all CORS requests. In a production environment, you will likely want to remove this or set it to something
+// more restrictive.
+app.use(cors({ origin: '*' }));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
 
-// Use your own routes here.
+// TODO: Set any custom routes needed here.
```

Comment:
Most examples should just need to add a couple routes to /api and not have to add anything here I would think now that I look at it again.

Reply:
I'm not sure if you're just making a comment or proposing I change anything.

Reply:
I'm not sure if you're just making a comment or proposing I change anything.

Reply:
Just a comment.